### PR TITLE
(IAC-144) - Removal of Inappropriate Terminology

### PR DESCRIPTION
**Pull request by david22swan**
Removal of any inappropriate or offensive terminology from the code base

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[![Build status](https://ci.appveyor.com/api/projects/status/v7w84n21w9yjspq7/branch/master?svg=true)](https://ci.appveyor.com/project/puppetlabs/pdk-planning/branch/master)
+[![Build status](https://ci.appveyor.com/api/projects/status/v7w84n21w9yjspq7/branch/main?svg=true)](https://ci.appveyor.com/project/puppetlabs/pdk-planning/branch/main)
```
Comment (pmcmaw) [inline README.md:1]: `https://ci.appveyor.com/project/puppetlabs/pdk-planning/branch/main` does not exist.

File: RFCs/0001-add-puppet-dev-flag.md [hunk 2]
```diff
@@ -20,15 +20,15 @@ Implement a new flag, `--puppet-dev`, to the `pdk test unit` and `pdk validate`
 
 - `pdk validate --puppet-dev`
 
-  Validates metadata and puppet code against the Puppet source on the `master` branch on GitHub.
+  Validates metadata and puppet code against the Puppet source on the `masmainter` branch on GitHub.
```
Comment (pmcmaw) [inline RFCs/0001-add-puppet-dev-flag.md:23]: typo

**Review by pmcmaw — COMMENTED** [review]
A couple of really minor things.

File: README.md (hunk 1 above)
Comment (david22swan) [reply README.md:1]: Only just created the main branch, it should be created soon

File: RFCs/0001-add-puppet-dev-flag.md (hunk 2 above)
Comment (david22swan) [reply RFCs/0001-add-puppet-dev-flag.md]: Fixed

File: README.md (hunk 1 above)
Comment (david22swan) [reply README.md:1]: that should appear when appveyor runs

**Comment by david22swan** [comment]
Have reopened a fresh pr, messed this one up

**Pull request closed without merging**